app/helpers/invitations_helper.rb

```diff
@@ -1,3 +1,6 @@
+# frozen_string_literal: true
+
+# TODO: remove and use presenter
```

Comment:
If there are only two small methods left here, why not move them to the presenter directly instead of leaving a TODO?